src/cursor/cur_layered.c

```diff
@@ -317,6 +319,11 @@ __clayered_open_stable(WT_CURSOR_LAYERED *clayered, bool leader)
 
         /* Layered cursor is not compatible with cursor_copy config. */
         F_CLR(clayered->stable_cursor, WT_CURSTD_DEBUG_COPY_KEY | WT_CURSTD_DEBUG_COPY_VALUE);
+
+        /* Update the stable table checkpoint timestamp. */
+        WT_ASSERT(
+          session, __wt_atomic_load64(&S2BT(session)->ckpt_timestamp) <= last_ckpt_timestamp);
+        __wt_atomic_store64(&S2BT(session)->ckpt_timestamp, last_ckpt_timestamp);
```

Comment:
Can we have a local btree variable?

Reply:
This logic is not a part of this function anymore.